admin/attributes_controller.php

```diff
@@ -758,116 +760,123 @@ function popupWindow(url) {
       if ($action == 'delete_all_attributes_confirm') {
         ?>
         <div class="row">
-          <?php echo zen_draw_form('delete_all', FILENAME_ATTRIBUTES_CONTROLLER, 'action=delete_all_attributes'); ?>
-          <?php echo zen_draw_hidden_field('products_filter', $_GET['products_filter']); ?>
-          <?php echo zen_draw_hidden_field('current_category_id', $_GET['current_category_id']); ?>
-          <div class="col-xs-6 col-sm-4 text-danger"><strong><?php echo TEXT_DELETE_ALL_ATTRIBUTES . $products_filter . '<br>' . zen_get_products_name($products_filter); ?></strong></div>
+          <?php
+          echo zen_draw_form('delete_all', FILENAME_ATTRIBUTES_CONTROLLER, 'action=delete_all_attributes');
+          echo zen_draw_hidden_field('products_filter', $_GET['products_filter']);
+          echo zen_draw_hidden_field('current_category_id', $_GET['current_category_id']);
+          ?>
```

Comment:
Convert 'old' lines 761 to 763 to use short-echo tags, for conformity.

Reply:
I agree on the 'not B&W'; let's see if anyone else chimes in.  If not, I'll let these go.